xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/action-menus.less

```diff
@@ -80,6 +80,37 @@
   }
 }
 
+ul {
+  @media (max-width: @screen-sm-max) {
+    // On mobile, do not show the avatar nor the notification buttons when the search input is open
```

Comment:
Maybe a dummy question, but in your comment you're talking about "not showing" and in the CSS you actually only move them: why not using a display none in that case? Or even a hidden class?